src/api/comments.ts

```diff
@@ -0,0 +1,13 @@
+import { request } from './api';
+
+export const getPostComments = (postId: number) => request(`/comments?postId=${postId}`);
+
+export const deletePostComments = (commentId: number) => request(`/comments/${commentId}`, { method: 'DELETE' });
+
+export const commentPostComments = (comment: Comm) => request('/comments', {
```

Comment:
```suggestion
export const commentPostComments = (comment: Comment) => request('/comments', {
```
Не сокращай названия